bundles/org.openhab.binding.cul.max/README.md

```diff
@@ -0,0 +1,204 @@
+| actual_temp     | Number:Temperature | This channel indicates the measured temperature of a thermostat (see below for more details).                                                                                                                                                             | thermostat, thermostatplus, wallthermostat                            |
+| valve           | Number             | This channel indicates the valve opening in %. Note this is an advanced setting, normally not visible.                                                                                                                                                    | thermostat, thermostatplus, wallthermostat                            |
+| locked          | Contact            | This channel indicates if the thermostat is locked for adjustments (OPEN/CLOSED). Note this is an advanced setting, normally not visible.                                                                                                                 | thermostat, thermostatplus, wallthermostat                            |
+| display_actual_temp | Switch         | Turns on / off if the thermostat displays the actual temperature instead of the set temperature                                                                                                                                                           | wallthermostat                            |
+| contact_state   | Contact            | This channel indicates the contact state for a shutterswitch (OPEN/CLOSED).
+| credits         | Number             | This channel indicates the remaining credits. Due to regulatory compliance reasons in Europe, the cul is allowed to send at no more than 1% of the time, which sums up to 36 seconds per hour. Once the threshold has been reached, the cul stops sending commands for the remaining time of the hour.  | cul_max_bridge, cun_max_bridge |
+| led             | Switch             | Turns on / off the led on the CUL. This channel is write only and may indicate false values after restart as it assume the led is always after restart | cul_max_bridge, cun_max_bridge |
+| pair_mode       | Switch             | Turns on / off the pair_mode on the MAX!CUL. It will switch to off after 1 minute | maxcul_bridge |
+| listen_mode     | Switch             | Turns on / off the listen_mode. In listen mode all messages are just logged. | maxcul_bridge |
+
+## Status
+
+The binding is currently in beta and it is recommended that you only use it expecting there to be bugs and issues. It is has enough features to be useful as a heating system, though lacks some of the finer features. This page will be updated as things progress.
+
+## Tutorial
+
+There was a tutorial using a Raspberry Pi with OH1 available at [technpol](https://technpol.wordpress.com/2016/04/09/configuration-of-maxcul-and-cul-dongle/), which addresses some of the configuration issues.

```

Comment:
Is it still applicable for OH3?

Reply:
The first half about preparing the cul  is still valid, the way how to write items configuration and sitemaps changed from OH1 to OH3.